trl/trainer/ppo_config.py

```diff
@@ -99,6 +101,7 @@ def __init__(
         accelerator_kwargs: Optional[dict] = {},
         tracker_project_name: Optional[str] = "trl",
         max_grad_norm: Optional[float] = None,
+        optimize_cuda_cache: Optional[bool] = False,
```

Comment:
are there drawbacks to setting it to true?

Reply:
also the order in the docstring and the kwargs is different, i think it's better to be consistent :)

Reply:
Fixed the order! 
The drawback is maybe about the computational time of the `step` function